ext/openssl/tests/openssl_error_string_basic.phpt

```diff
@@ -94,8 +94,7 @@ while (($enc_error_new = openssl_error_string()) !== false) {
 var_dump($error_queue_size);
 echo "\n";
 
-$is_111 = OPENSSL_VERSION_NUMBER >= 0x10101000;
-$err_pem_no_start_line = $is_111 ? '0909006C': '0906D06C';
+$err_pem_no_start_line = '0909006C';
```

Comment:
you can probably just not use variable in this case...

Reply:
This `$err_pem_no_start_line` is used in the test file five times. There is also a `openssl_error_string_basic_openssl3` with a similar variable used a few times in that test file too.

Given the magic nature of the variable, I'm leaning to keep this variable. What do you think?

Reply:
That variabable was mainly added for this switched logic but it's fine if you keep it as it is. It won't do any harm and might make it a bit clearer.